tutorials/cookbook/Recipes Thomas.ipynb

```diff
@@ -0,0 +1,311 @@
+{
```

Comment:
<p>I think this is exactly what<code> H1 &lt;&lt; H2</code> does (lshift).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/xgi-org/xgi/pull/464/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<pre class="ql-syntax" spellcheck="false">&gt;&gt;&gt; H1 = xgi.Hypergraph([[1,2,3],[3,4]])
&gt;&gt;&gt; H2 = xgi.Hypergraph([[1,2],[3,4],[4,5,6]])
&gt;&gt;&gt; h = H1 &lt;&lt; H2
&gt;&gt;&gt; print(h.edges.members())
[{1, 2, 3}, {3, 4}, {1, 2}, {3, 4}, {4, 5, 6}]

</pre><p>You see that the edges that is present in both hypergraphs is present twice in the merged hypergraph</p>